chapters/operatorsandexpressions.tex

```diff
@@ -1235,7 +1235,8 @@ \subsection{Event-Related Operators with Function Syntax}\label{event-related-op
 equation
   x = if time < 1 then 2 else time - 2;
   z = smooth(0, if time < 0 then 0 else time);
-  y = smooth(1, noEvent(if x < 0 then 0 else sqrt(x) * x)); // noEvent is necessary.
+  y = smooth(1,
```

Comment:
This seems somewhat spurious.
Was the line too long?

Reply:
Yes.